corehq/apps/events/views.py

```diff
@@ -406,6 +406,8 @@ def page_context(self):
             form = EditAttendeeForm(domain=self.domain, instance=instance)
         context.update({
             'attendee_id': self.attendee_id,
+            'attendee_name': instance.name,
+            'attendee_active_event_count': instance.active_event_count(),
```

Comment:
This feels expensive to calculate -- it doesn't scale well for large numbers of attendees, especially not if combined with large numbers of events. What do you think about only checking if the user tries to delete the attendee?

Reply:
I could do that, but then we lose having the warning alert on the edit page indicating why the attendee cannot be deleted. One of the requirements of the ticket was to grey out the delete button and show a message as to why it's greyed out. Should we rather not disable the button and instead let the user delete but just return with an error message if the delete action is not allowed?

Reply:
After having reworked the function to determine if an attendee is tracked, there is a smaller performance impact now and it should be okay to call when getting the page context. What are your thoughts @kaapstorm?